src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Provides extra information about a section item (such as video length, PDF, etc). Shown in the upper right corner.</inlineHelpText>
+        <label>Extra Info Label</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>GS_Admin_Checklist_Section__c</fullName>
+        <description>The section on the Get Started page that this item is part of.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The section on the Get Started page that this item is part of.</inlineHelpText>
+        <label>GS Admin Checklist Section</label>
+        <referenceTo>GS_Admin_Checklist_Section__mdt</referenceTo>
+        <relationshipLabel>GS Admin Checklist Items</relationshipLabel>
+        <relationshipName>GS_Admin_Checklist_Items</relationshipName>
+        <required>true</required>
+        <type>MetadataRelationship</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Has_Link__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates if the item has a link to display.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Indicates if the item has a link to display.</inlineHelpText>
+        <label>Has Link</label>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Image__c</fullName>
+        <description>The path and filename of the image in the Static Resource to render.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The path and filename of the image in the Static Resource to render.</inlineHelpText>
+        <label>Image</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Link_Label__c</fullName>
+        <description>The api name of the label to use for a link on the Get Started page.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The api name of the label to use for a link on the Get Started page.</inlineHelpText>
+        <label>Link Label</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Link_URL__c</fullName>
+        <description>he target URL for a link on the Get Started page.</description>
```

Comment:
There is a typo in the description and inlineHelpText. Please add a "T" to the beginning of both labels.